### ISSUE TITLE
add a :sessiondelete command

### ISSUE THREAD

**Issue by aleprovencio** [gwash/penta-sessions#4]
Perhaps along with the mapping 'sd' following the existing 'sl', 'ss' and 'sa'?

**Comment by gwash** [comment]
Well, this could be a little dangerous, since I cannot be sure it's only deleting session files, I could add some confirmation though.

**Comment by aleprovencio** [comment]
Yeah, that would be cool. This way your plugin would behave as vim's sessionman
and like, where you don't need to leave the app and open file manager to delete
the session file.
